### ISSUE TITLE
router-link propagates click

### ISSUE THREAD

**Issue by christianw** [Zulko/eagle.js#59]
Before anything else: eagle.js is a fantastic framework!

Having mouseNavigation enabled and using router-link in a template to redirect to another slide, the click is recorded again on the new slide and the next step (in my case the next slide) is shown. I could not find a way to fix this.

Regards
Christian

**Comment by yaodingyd** [comment]
@christianw  can you provide a sandbox link?

**Comment by christianw** [comment]
Thanks for the quick answer. It happens (for me) with every <router-link> to another slide. The correct slide gets activated, the the click is again captures on this slide and the next slide is shown.

I managed to solve it, though, by using <router-link v-on:click.native.stop.prevent> and in this way stopping event propagation. Maybe that's the best way for now and perhaps it could be added in the documentation.

I have made a few other changes like using also ArrowUp/Down and PageUp/Down/Space for navigation to mimick the legacy Powerpoint. I can create a pull request but the changes are trivial anyway.

Again: great fun to work with this!
